contracts/WildcardSteward_v2.sol

```diff
@@ -220,28 +236,26 @@ contract WildcardSteward_v2 is Initializable {
         }
     }
 
-    function fixTotalPatronTokenGenerationRate(uint256[] memory tokens)
+    function upgradeToV3(uint256[] memory tokens, address _withdrawCheckerAdmin)
         public
         onlyAdmin
     {
+        require(withdrawCheckerAdmin == address(0));
```

Comment:
This is to ensure that the function can only be called once to change the withdrawCheckerAdmin and intialize?

Reply:
Yeah, it isn't super necesary since it is `onlyAdmin`, I just thought why not :)